bin/epispot

```diff
@@ -8,7 +8,12 @@ except:
     print("Please install Fire to use epispot's CLI")
     print('>>> pip install fire')
 
+
 class bcolors:
+    """
+    Styling Format:
+        C[color/format/style][additional 2 for a new ANSI code with the same style/color]
```

Comment:
What is the 2 for if there is already a duplicate ANSI code?

Reply:
Yes, that's what it states in the docstring.

Reply:
**What is a new ANSI code?**
**What even is an ANSI code??**
**Why would you want two of the exact same style???**

Reply:
ANSI escape codes are a way for getting string styling. See [this](https://notes.burke.libbey.me/ansi-escape-codes/) for more.

As for why you'd want two of the same style, it is possible a certain OS might not support a ANSI escape code, therefore having two might be useful.